terraform/azure/infra/variables.tf

```
@@ -20,6 +20,12 @@ variable "deployment_is_private" {
   default     = false
 }
 
+variable "use_web_application_firewall" {
+  description = "If true, we'll provision a web application firewall for increased security"
+  type        = bool
+  default     = false
```

Should we keep the previous secure default here by setting use_web_application_firewall default value to true in the variable?
```suggestion
  default     = true
```

---

I'm 50/50 on this one.  I setup WAF because it felt more "Secure" but it's expensive and pretty false positive prone.  I think the better default is to use a regular load balancer.